lib/jekyll-titles-from-headings/generator.rb

```diff
@@ -32,8 +34,12 @@ def initialize(site)
 
     def generate(site)
       @site = site
+      return if disabled?
 
-      site.pages.each do |document|
+      pages = site.pages
+      pages = site.pages + site.documents if collections?
```

Comment:
Semantically, I'd call this variable documents, especially since we call it documents in the iterator below to make the meaning more clear (pages feel like documents to me, more so than documents feel like pages).

Reply:
Also, should `site.documents` be `site.docs_to_write`?

Reply:
Yes, that name was chosen without much thought. Will change 😄 

Also `site.docs_to_write` is the one to use.